tests/Unit/Helpers/Evolution/DiscontinuousGalerkin/BoundaryCorrections.hpp

```diff
@@ -44,6 +44,45 @@ struct inverse_spatial_metric_tag<true> {
   template <typename System>
   using f = typename System::inverse_spatial_metric_tag;
 };
+
+template <size_t Dim>
+void adjust_inverse_spatial_metric(
+    const gsl::not_null<tnsr::II<DataVector, Dim>*> inv_spatial_metric) {
+  // Make sure the inverse spatial metric is delta_{ij}+something smaller
+  // so that it's physically reasonable.
```

Comment:
[minor] Now that this is a helper function, can you expand the comment a bit (and maybe move it outside the function) to better document that it expects random numbers in [0,1] as inputs with which to compute the output delta_ij + small?